modules/videopress/shortcode.php

```diff
@@ -150,7 +150,7 @@ function videopress_handle_editor_view_js() {
 
 	add_action( 'admin_print_footer_scripts', 'videopress_editor_view_js_templates' );
 
-	wp_enqueue_script( 'videopress-editor-view', plugins_url( 'js/editor-view.js', __FILE__ ), array( 'wp-util', 'jquery' ), false, true );
+	wp_enqueue_script( 'videopress-editor-view', plugins_url( 'js/editor-view.js', __FILE__ ), array( 'wp-util', 'media-editor', 'jquery' ), false, true );
```

Comment:
I'll retry my plugin with the dependency fix, if it works, then my sole reason for caring goes away, might be easier for us both. :+1: